scripts/downloadIpfsDaemon.js

```diff
@@ -9,6 +9,55 @@ const mkdirp = require('mkdirp')
 const path = require('path')
 const util = require('../lib/util')
 const ipfsVersion = '0.9.0'
+const fsRepoVersion = '1.0.0'
+
+// Downloads the current (platform-specific) fs-repo-migration component
+const downloadFsRepoMigration = (platform) => {
+  if (platform !== 'darwin')
+      return
+  const repoPath = path.join('build', 'ipfs-daemon-updater', 'downloads')
+
+  const ipfsDistPrefix = `https://dist.ipfs.io/fs-repo-10-to-11/v${fsRepoVersion}/`
+
+  const zipSuffix = '.tar.gz'
+
+  const repoFilename = `fs-repo-10-to-11_v${fsRepoVersion}_${platform}-amd64`
+  const targetFilename = `fs-repo-10-to-11`
+  const ipfsURL = ipfsDistPrefix + repoFilename + zipSuffix
+
+  let sha512IPFS = ''
+
+  switch (platform) {
+    case 'darwin':
+      sha512IPFS = '9a8b8b18c7757e5f2a0abd62dc33fc14616195b27776a967d4adff361a2becedebf286eee156af05bdbf1f39b0da5cc33264939a695a7866abe50a06b817d5cd'
+      break
+    default:
+      throw new Error('fs-repo migration tool download failed; unrecognized platform: ' + platform)
```

Comment:
Won't this break Windows and Linux?

Reply:
This is script for out build agent, imho better to throw error immediatelly if we see wrong usage because it will be harder to figure out the reason later

Reply:
Ack. Does this mean the additional binary will only be included on macOS?

Reply:
yes